<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tabela">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>841</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>139</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Arquivo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAbrir">
       <property name="text">
        <string>Abrir</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>391</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Ordenacao</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxOrdenacao">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string> Nenhum</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string> Nome</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string> Matricula</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string> Departamento - Nome</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string> Titulação - Nome</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string> Tipo de Contrato - Nome</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string> Departamento - Titulação - Nome</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string> Departamento - Tipo de Contrato - Nome</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAttOK">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>417</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Busca Sequencial - Nome</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Busca Sequencial - Matricula</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Busca Binaria - Nome</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Busca Binaria - Matricula</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditDados"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAtt">
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>937</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
